ProQuest.js

```diff
@@ -120,21 +120,26 @@ function fetchEmbeddedPdf(url, item, callback) {
 }
 
 function getSearchResults(doc, checkOnly) {
-	var tabs = doc.getElementsByClassName('tabContent');
 	var root;
-	if (tabs.length) {
-		for (var i = 0; i < tabs.length; i++) {
-			if (tabs[i].offsetHeight) {
-				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
-				root = tabs[i].getElementsByClassName('resultListContainer')[0];
+	var tabs = doc.getElementsByClassName('tabContent');
+	for (var i = 0; i < tabs.length; i++) {
+		if (tabs[i].offsetHeight) {
+			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
+			root = tabs[i].getElementsByClassName('resultListContainer')[0];
+			break;
+		}
+	}
+	if (!root) {
+		var elements = doc.getElementsByClassName('resultListContainer');
+		for (var i=0; i<elements.length; i++) {
+			if (elements[i] && !elements[i].offsetHeight) {
+				root = elements[i];
 				break;
 			}
 		}
-	} else {
-		root = doc.getElementsByClassName('resultListContainer')[0];
-		if (root && !root.offsetHeight) return false; // Not sure if this can actually happen
+		//root = doc.getElementsByClassName('resultListContainer')[0];
+		//if (root && !root.offsetHeight) return false; // Not sure if this can actually happen
 	}
-
 	if (!root) return false;
```

Comment:
@symac Please look carefully at the logic here for `root`. AFAIK the variable is not used any further but it has to be here the search results (was this because of the ebrary results?).